org.wxhexeditor.wxHexEditor.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.wxhexeditor.wxHexEditor",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "wxHexEditor",
+    "rename-icon": "wxHexEditor",
+    "rename-desktop-file": "wxHexEditor.desktop",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--filesystem=home",
+        "--device=all"
+    ],
+    "modules": [
+        {
+          "name": "wxWidgets",
+          "cleanup": [
+              "/bin",
+              "/include",
+              "/share/bakefile",
+              "/share/aclocal",
+              "/lib/wx/include"
+          ],
+          "config-opts": [
+              "--with-gtk=3",
+              "--disable-sdltest",
+              "--enable-unicode",
+              "--enable-display",
+              "--enable-propgrid",
+              "--disable-webkit",
+              "--disable-webview",
+              "--disable-webviewwebkit",
+              "--with-expat=builtin",
+              "--with-libiconv=/usr"
+          ],
+          "build-options" : {
+              "cxxflags":"-std=c++0x"
+          },
+          "sources": [ 
+            {  
+                "type": "archive", 
+                "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2",
+                "sha256":"96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
+            }
+          ]
+        },
+        {
+            "name": "wxHexEditor",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make -j $FLATPAK_BUILDER_N_JOBS",
+                "install -D -m 755 wxHexEditor /app/bin/wxHexEditor",
+                "install -D -m 644 resources/wxHexEditor.png /app/share/icons/hicolor/128x128/apps/wxHexEditor.png",
+                "desktop-file-edit --remove-key=Keywords --set-key=Keywords --set-value=disk\\;drive\\;volume\\;harddisk\\;hdd\\;disc\\;cdrom\\;dvd\\;partition\\;iso\\;image\\;backup\\;restore\\;editor\\; --set-key=Categories --set-value=Utility\\;TextEditor\\; resources/wxHexEditor.desktop",
```

Comment:
```suggestion
                "desktop-file-edit --remove-key=Keywords --set-key=Keywords --set-value='disk;drive;volume;harddisk;hdd;disc;cdrom;dvd;partition;iso;image;backup;restore;editor;' --set-key=Categories --set-value='Utility;TextEditor;' resources/wxHexEditor.desktop",
```
This should work.

Reply:
oops, i thought json.. that flew right past me